examples/pytorch/01-Getting-started.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
This is something @edknv suggested, it ensures that the background thread gets removed. I am working on a way to see if we can move this inside our model/trainer code. Because I think the context manager approach works for single GPU, but I don't think it will work in a multi-GPU setting.